server/controllers/cartItems-controller.js

```diff
@@ -48,7 +49,7 @@ const createCartItem = async (req, res) => {
 const updateCartItem = async (req, res, next) => {
     const itemToUpdate = await models.Cart_Item.findOne({
         where: {
-            "product_uuid": req.body.item.product_uuid
+            "uuid": req.body.item.uuid
```

Comment:
So is this a switch in naming or in usage?
The payload is send a product_uuid or an item_uuid?

ideally, all you'd need is the item_uuid.